lib/codebreaker/validator.rb

```diff
@@ -0,0 +1,42 @@
+require_relative './errors/validator_error'
+require_relative './constants'
+
+module CodeBreaker
+  class Validator
+    def self.validates_name?(name)
```

Comment:
redo your methods with `?`. Such methods must return `true/false` but in your case it returns true or raises an error. Remove `?` in the names of the methods and return nil in case of success